core/include/kernel/user_access.h

```diff
@@ -33,6 +33,21 @@ static inline TEE_Result copy_from_user(void *kaddr __unused,
 TEE_Result copy_to_user_private(void *uaddr, const void *kaddr, size_t len);
 TEE_Result copy_to_user(void *uaddr, const void *kaddr, size_t len);
 
+#define get_user(_x, _p) ({						\
+	TEE_Result __res;						\
```

Comment:
The coding style requires local variables to be initialized when declared, `0` is fine.

Reply:
Use upper case names for macro: `GET_USER()`, `PUT_USER()`.
Implementation should check type of `_x` and `_p` match is size.
Caller shall check the return values.

Reply:
Added initial value

Reply:
Added compile-time assertion